owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewVideoActivity.kt

```diff
@@ -92,7 +97,7 @@ class PreviewVideoActivity : FileActivity(), Player.Listener, OnPrepareVideoPlay
     private var playWhenReady = true
     private var playbackPosition: Long = 0
     private var windowInsetsController: WindowInsetsControllerCompat? = null
-    private val previewVideoViewModel: PreviewVideoViewModel by viewModel()
+    private val previewVideoViewModel: PreviewVideoViewModel by viewModel() { parametersOf(intent.getParcelableExtra(EXTRA_FILE, OCFile::class.java)) }
```

Comment:
Can't you extract it from `arguments` in this case? Is it necessary to explicitly indicate the class type as second parameter?

Reply:
Changed. It's no necessary the second parameter.